src/main/java/be/feelio/mollie/Client.java

```diff
@@ -10,14 +10,17 @@
     @Getter
     private final String endpoint;
 
+    @Getter
+    private final Config config = new Config();
```

Comment:
I Think it would be a bit cleaner if we initialize the config object in the constructor